CMakeLists.txt

```diff
@@ -13,8 +13,12 @@
 cmake_minimum_required(VERSION 2.8.4 FATAL_ERROR)
 cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION})
 
+IF (NOT DEFINED KFS_DIR_PREFIX)
+    set (KFS_DIR_PREFIX ${CMAKE_SOURCE_DIR})
```

Comment:
KFS_DIR_PREFIX is intended for other projects to use QFS build within another cmake build, that has its now top level cmake, and relies on QFS code. Presently Quantcast internal project relies on this variable.